package/MDAnalysis/analysis/dssp/pydssp_numpy.py

```diff
@@ -63,7 +63,7 @@ def _upsample(a: np.ndarray, window: int) -> np.ndarray:
     return _unfold(_unfold(a, window, -2), window, -2)
 
 
-def _unfold(a: np.ndarray, window: int, axis: int):
+def _unfold(a: np.ndarray, window: int, axis: int):  # pragma: no cover
```

Comment:
Did you add `no cover` because you don't have an explicit test or because there's no coverage. If it gets covered as part of testing other code then that's fine. 

But if we don't need it then we should just remove it.

Reply:
Same.